src/lib/twitch.ts

```diff
@@ -0,0 +1,96 @@
+import config from './config';
```

Comment:
Invalid indentation

Reply:
Don't see what's wrong here, import is copied from database.ts in the same dir

Reply:
It was more a comment on the whole file, it uses 2 space indentation instead of tabs